tests/app/parser/test_mci_mini.py

```diff
@@ -0,0 +1,72 @@
+from app.parser.schema_parser_factory import SchemaParserFactory
+from app.parser.schema_parser import SchemaParser
+from app.model.questionnaire import Questionnaire
+from app.model.group import Group
+from app.model.block import Block
+from app.model.section import Section
+from app.model.question import Question
+from app.model.response import Response
+import os
+import json
+
+
+def test_mci_mini():
```

Comment:
You could use the Python unit test framework rather than just using assert statements